docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -42,99 +42,98 @@ rg.init(workspace="my_shared_workspace")
 rg.set_workspace("my_private_workspace")
 ```
 
-### *users.yml*
-
-The above user management model is configured using a YAML file which server maintainers can define before launching a Argilla instance.
-This can be done when launching Argilla from Python or with the provided `docker-compose.yml`. Read below for more details on the different options.
-
 ## Default user
 
-By default, if you don't configure a `users.yml` file, your Argilla instance is pre-configured with the following default user:
+By default, if the Argilla instance has no users, following default admin user will be configured:
 
 - username: `argilla`
 - password: `1234`
 - api_key: `argilla.apikey`
 
-for security reasons we recommend changing at least the password and API key.
+for security reasons we recommend changing at least the password and API key. You can configure
+the default user as follow:
+
+```bash
+python -m argilla.tasks.users.create_default --password changeme --api-key new-api-key
+```
+
 
 :::{note}
 To connect to an old Argilla server using client `>=1.3.0`, you should specify the default user API key `rubrix.apikey`.
```

Comment:
Shouldn't `>=1.3.0` be `<=1.3.0` instead?

Reply:
I see what you mean. I  believe the text is correct, but feel contradictory when reading.